editoast/src/views/test_app.rs

```diff
@@ -135,6 +140,16 @@ impl TestAppBuilder {
         self
     }
 
+    pub fn with_trains_traffic(mut self, trains_traffic: Vec<TrainTraffic>) -> Self {
+        for train in trains_traffic {
+            tracing::debug!(train.name = train.train_name, "Loading train traffic");
+            if let Err(e) = self.trains_traffic.add_train_traffic(train) {
+                tracing::error!("Failed to add train traffic: {:?}", e);
```

Comment:
```suggestion
            if let Err(e) = self.trains_traffic.add_train_traffic(train) {
                panic!("Failed to add train traffic: {e:?}");
```

this is a test setup failure, we want to be noticed right away